src/applications/static-pages/events/components/Results/ResultsWhereContent.jsx

```diff
@@ -2,20 +2,36 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { deriveEventLocations } from '../../helpers';
 
-const WhereContent = ({ fieldLocation, fieldType, derivedLocations }) => {
+const WhereContent = ({
+  event,
+  fieldLocation,
+  fieldType,
+  derivedLocations,
+}) => {
   if (fieldType === 'online') return 'This is an online event.';
+  const locationAddress =
+    derivedLocations?.length > 0 ? derivedLocations.join(' ') : '';
   return (
     <>
       <a href={fieldLocation?.entity?.entityUrl.path}>
         {fieldLocation?.entity?.title}
       </a>
+      <p className="vads-u-margin--0" key={location}>
+        {event?.fieldLocationHumanreadable}
```

Comment:
Is this field often not defined? If so, could we make rendering the paragraph conditional upon the existence of this field?

Reply:
Good catch